.gitignore

```diff
@@ -31,4 +31,11 @@
 *.out
 *.app
 
-build/*
\ No newline at end of file
+# Files and Directories
+build/*
+.vscode/*
+.github/*
+.cache/*
+
+run.sh
+CMakeCache.txt
```

Comment:
But why does it even exist? To me it looks like someone configured CMake in project's root directory instead of `/build` and didn't remove this file thinking it was relevant.

Reply:
Okay, I shall delete it and remove it from the `.gitignore`